husky_gazebo/controller/joint_state_controller.yaml

```diff
@@ -0,0 +1,3 @@
+joint_state_controller:
+    type: joint_state_controller/JointStateController
+    publish_rate: 50  
```

Comment:
Please use the one from husky_control instead: https://github.com/husky/husky_control/tree/indigo-devel/config